flow/connectors/postgres/postgres_cdc_test.go

```diff
@@ -603,36 +604,36 @@ func (suite *PostgresCDCTestSuite) TestAllTypesHappyFlow() {
 	suite.Equal(&protos.TableSchema{
 		TableIdentifier: allTypesHappyFlowSrcTableName,
 		Columns: map[string]string{
-			"id":  model.ColumnTypeInt64,
-			"c1":  model.ColumnTypeInt64,
+			"id":  string(qvalue.QValueKindInt64),
+			"c1":  string(qvalue.QValueKindInt64),
 			"c2":  model.ColumnHexBit,
```

Comment:
why are these still `model.*` ?

Reply:
there isn't any implementation for some types currently, I left them as placeholders so we can replace with new types once implemented